pkg/cmd/kafka/describe/describe.go

```diff
@@ -125,6 +127,11 @@ func runDescribe(opts *options) error {
 		}
 	}
 
+	if opts.bootstrapServer {
+		fmt.Fprintln(opts.IO.Out, kafkaInstance.GetBootstrapServerHost())
+		return nil
+	}
```

Comment:
I think we should be checking if the bootstrap_server_host exists, and if it does not print some info message letting the user know this.

Reply:
Agree. We can extract this code into helper: 
https://github.com/redhat-developer/app-services-cli/blob/main/pkg/connection/keycloak_connection.go#L254-L267

Reply:
I've added a message, please take a look.